### ISSUE TITLE
Conversation to USWDS v3 - Text area - Documentation

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#1497]
## Description
Update documentation and guidance for the Textarea component on design.va.gov based on guidance from USWDS v3.

## Details
[USWDS Textarea which is a variation of textinput  component](https://designsystem.digital.gov/components/text-input/)
[VA DS Textarea  component](https://design.va.gov/components/form/textarea)

## Tasks
- [ ] Update component documentation for design.va.gov
- [ ] Request documentation draft review
- [ ] Once documentation is approved, publish to design.va.gov

## Acceptance Criteria
- [ ] Component documentation is published on design.va.gov

**Comment by humancompanion-usds** [comment]
This has been done.

This issue has been inactive for over two years and appears to be superseded by more recent events and updates. To keep our backlog manageable and relevant, we are closing this issue. If you believe this issue is still relevant and requires further attention, please feel free to reopen it or create a new issue with updated details.

Thank you for your understanding and contributions!
